utils/webhook/github.handler.ts

```diff
@@ -340,6 +340,41 @@ export async function githubWebhookHandler(
             }
         );
 
+        /** In case if user has added some other labels on issue */
+        const githubLabels = issue.labels.filter(
+            label => label.name !== "linear"
+        );
+
+        const linearCreatedLabels: Array<string> = [];
+        for (const githubLabel of githubLabels) {

```

Comment:
Could you reduce this loop to a single API call by filtering for multiple labels at the same time in `filter: { name: { ... } ... }` below?